lib/user.js

```diff
@@ -98,15 +102,15 @@ const instanceMethods = {
   },
 
   // Internal helpers.
-  _functionsOnService(service) {
+  _functionsOnService(serviceName) {
     const user = this;
     return new Proxy(
       {},
       {
         get(target, name, receiver) {
           if (typeof name === "string" && name != "inspect") {
             return (...args) => {
-              return user.callFunction(name, args, service);
+              return user._callFunctionOnService(name, serviceName, args);
```

Comment:
I think these arguments should be spread into the call.

```suggestion
              return user._callFunctionOnService(name, serviceName, ...args);
```

Reply:
Good catch.  We also needed to return the call to  `_callFunctionOnService`.  Jenkins tests should pass now